python_modules/libraries/dagster-mlflow/dagster_mlflow/resources.py

```diff
@@ -17,20 +17,20 @@
 CONFIG_SCHEMA = {
     "experiment_name": Field(str, is_required=True, description="MlFlow experiment name."),
     "mlflow_tracking_uri": Field(
-        Noneable(str),
+        Noneable(str),  # type: ignore
```

Comment:
whats going on here with `Noneable`?

Reply:
@alangenfeld All good catches, all fixed now. The type ignores were leftover and unnecessary, the gql change was also a mistake.